app/services/alert_image_generator.rb

```diff
@@ -0,0 +1,46 @@
+module AlertImageGenerator
+  BASE_TEMPLATE_PATH = Rails.root.join("app", "assets", "images", "premium_alert_template.png")
```

Comment:
Could we name this "promoted_alert_template" to go with our new name?

Also, does it matter that we're translating this to a jpg? Would it be faster/better to make this a jpg to begin with?

Reply:
My hunch is that intermediate conversion might degrade the quality without any benefit. The final product is saved as a JPG to reduce the file save though.